spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -165,15 +160,55 @@ def then_i_should_see_correct_breakdown_summary(npq_lead_provider, npq_contract)
       .to have_sibling("dd.govuk-summary-list__value", text: ParticipantDeclaration::NPQ.submitted_for_lead_provider_and_course(npq_lead_provider, npq_contract.course_identifier).count)
   end
 
-  def then_i_should_see_correct_payment_breakdown(npq_contract)
-    within "table.govuk-table tbody tr.govuk-table__row:nth-child(1)" do
-      expected_per_participant_portion = npq_contract.per_participant * npq_contract.service_fee_percentage / (100 * npq_contract.service_fee_installments)
+  def expected_per_participant_service_fee_portion(npq_contract)
+    npq_contract.per_participant * npq_contract.service_fee_percentage / (100 * npq_contract.service_fee_installments)
```

Comment:
I think `expected_service_fee_portion_per_participant` reads much better but I know the previous method was worded similarly